testsuite/pytests/test_spatial/test_basics.py

```diff
@@ -421,6 +469,82 @@ def test_GetTargetPositions(self):
             self.assertAlmostEqual(positions[indx][0], p[0][indx][0])
             self.assertAlmostEqual(positions[indx][1], p[0][indx][1])
 
+    @unittest.skipIf(not HAVE_NUMPY, 'NumPy package is not available')
+    def test_GetSourcePositions(self):
+        """Test that GetSourcePosition works as expected"""
+
+        cdict = {'rule': 'pairwise_bernoulli',
+                 'p': 1.}
+        sdict = {'synapse_model': 'stdp_synapse'}
+
+        # For co-dependent properties, we use `set()` instead of kernel attributes
+        nest.set(use_compressed_spikes=False, sort_connections_by_source=False)
+
+        layer = nest.Create('iaf_psc_alpha',
+                            positions=nest.spatial.grid(shape=[1, 1],
+                                                        extent=(1., 1.),
```

Comment:
Why use a tuple for the extent here?

Reply:
No deep reason, just consistency with the other tests.